standards/documentation.md

```diff
@@ -0,0 +1,54 @@
+# Documentation Standards
+
+All coding projects MUST include and maintain minimum documentation standards.
+
+## Minimum Standards
+
+Applications MUST include a README
```

Comment:
I think this is a great suggestion. Like @rhernand3z suggestion, maybe we can say it's RECOMMENDED (?) to create a Wiki for additional documentation.